src/app/keyboard/components/virtual-keyboard/virtual-keyboard.component.scss

```diff
@@ -28,15 +28,14 @@
   }
 }
 
-
 .key {
   display: inline-block;
   padding: 12px 20px;
   margin: 10px;
-  background-color: gray;
+  background-color: var(--dark-gray-color);
   border-radius: 4px;
   cursor: pointer;
-  box-shadow: 2px 2px 4px rgba(0,0,0,0.4);
+  box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.4);
```

Comment:
We can't use a variable here?

Reply:
@LindvedKrvang I only find the following 2 approaches for this:
1)
```
$black-color: #000000
box-shadow: 0px 2px 1px -1px rgba($black-color, 0.2); <== It's work

box-shadow: 0px 2px 1px -1px rgba(var(--black-color), 0.2); <== doesn't work
```
We need to keep it this new variables in different file` $black-color: #000000` 

```
2)
--black-color-with-opacity-02: rgba(0, 0, 0, 0.2);
box-shadow: 0px 2px 1px -1px var(--black-color-with-opacity-02)
```
Down side of this we need to keep all different opacities in separated values and we need to have 2 different color representations HEX and rgba

3)
We can leave this values without variables and hardcoded as colors now they are used in ~ 20 places

Reply:
Done with approaches #2